rental_pricelist/models/sale.py

```diff
@@ -277,8 +276,18 @@ def product_uom_change(self):
                             key
                         )
                         break
+        if self.number_of_time_unit:
+            self.product_uom_qty = self.rental_qty * self.number_of_time_unit
```

Comment:
It was already be done in function rental_qty_number_of_days_change()

Reply:
Without that line, update_prices method calculates incorrect product_uom_qty

Reply:
rental_qty_number_of_days_change changes product_uom_qty so the other onchange function will be triggered again with the correct product_uom_qty.

Reply:
This was correct, I deleted those lines in next commit